R/tbl_regression_methods.R

```diff
@@ -13,6 +13,21 @@
 #' @inheritParams tbl_stack
 NULL
 
+#' @export
+#' @rdname tbl_regression_methods
+tbl_regression.model_fit <- function(x, ...) {
+  tbl_regression(x = x$fit, ...)
+}
+
+#' @export
+#' @rdname tbl_regression_methods
+tbl_regression.workflow <- function(x, ...) {
```

Comment:
The only thing I noticed here is that when you use a workflow and get the error message and then follow the instructions to update the syntax, the results of tbl_regression() do not include variable labels that were included in the original data.

Reply:
@zabore thanks for taking a look!

I _think_ the labels are likely lost in the preprocessing that happens in the workflow....hmmm, I wonder if there is a way to get at the labels.....